mongo/collection.go

```diff
@@ -529,7 +564,19 @@ func (coll *Collection) updateOrReplace(ctx context.Context, filter bsoncore.Doc
 	return res, err
 }
 
-// UpdateOne updates a single document in the collection.
+// UpdateOne executes an update operation (https://docs.mongodb.com/manual/reference/command/update/) to update at
+// most one document in the collection.
+//
+// The filter parameter must be a document containing query operators and can be used to select the document to be
+// updated. It cannot be nil. An empty document (e.g. bson.D{}) will select one of the documents in the collection. If
```

Comment:
You can delete 

> An empty document (e.g. bson.D{}) will select one of the documents in the collection

The last sentence of the paragraph adequately specifies the behavior, and is the more useful case anyway, as it would be rare to actually specify an empty document for this method.

Reply:
deleted this sentence for all of the `*One` and `FindOneAnd*` methods.